manager/src/main/java/org/openremote/manager/setup/builtin/ArtNetDemoSetup.java

```diff
@@ -0,0 +1,93 @@
+package org.openremote.manager.setup.builtin;
```

Comment:
No other protocol uses a setup task to provision assets and provisioning is project specific and is always handled in the project code